src/ocl/clhelper.cpp

```diff
@@ -42,7 +43,7 @@
 
 namespace miopen {
 
-void ParseDevName(std::string& name)
+void WorkaroundIssue1711(std::string& name)
```

Comment:
Is this really a workaround?

Reply:
I think so. I consider this as a workaround for the quiet change in the OpenCL API, see original issue at https://github.com/AMDComputeLibraries/MLOpen/issues/1711.